src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/PrisonerServiceTest.kt

```diff
@@ -23,11 +24,12 @@ import uk.gov.justice.digital.hmpps.visitscheduler.dto.prison.api.PrisonerHousin
 import java.time.LocalDate
 
 @ExtendWith(MockitoExtension::class)
-class PrisonApiServiceTest {
+class PrisonerServiceTest {
 
```

Comment:
Can we have a unit test for hasPrisonerGotEnhancedPrivilege here for the following:

1. Tests for a null values
2. Tests for happy path
3. Tests for empty string 

May be i'm just missing it, unless I should just put this in the integration tests, which I think cover all, what do you think?

Reply:
Added the unit tests to test the service with different combinations as suggested.